### DISCUSSION TITLE
Windows Terminal Preview v1.19.3172.0

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#16315] [Announcements]
Welcome to the mid-November servicing release for Windows Terminal!

This release includes the following changes:

## Changes

* On newer version of Windows 11, the Store will no longer terminate Terminal to install updates (#16250)
* You can now set the opacity of a profile differently between its `focused` and `unfocused` appearances (#15974) (thanks @Jaswir!)

### Usability

* Environment variables will no longer spontaneously disappear if one happens to have a `NUL` in it (#16190)
* The "Azure Cloud Shell" connector has been updated to support the new cloud shell protocol (#16247)
* The regression where "Cascadia Mono" (or "... Code") would not be found on upgrade has been un-regressed (#16196)
* The light color schemes now include a selection background that is actually _visible_ (#16243) (thanks @TahaHaksal!)
   * You may need to delete the light color schemes from your settings to see this change.
* With the new rendering engine, some pixel shaders should no longer cause characters to have a black background (#16219)
* Screen readers and prompt marks now work more correctly with the newly-rewritten "cooked" input handler (#16105)
* The new "cooked" input handler no longer misprints parts of the input when you fail a tab completion (#16273)

### Reliability

* Terminal no longer leaks nigh-unto infinite amounts of memory when it's out of focus (#16251)
* When you close a window _on Windows 10_, there's a much lower chance of Terminal blowing up spontaneously (#16267)
* Toggling the alternate screen buffer in a `PROCESSED_INPUT` console application will no longer crash the console (#16187)
* The throughput of pasted text, which was reduced by contention on the I/O interface for the console, has been improved (#16224)

### Miscellaneous

* Scrollbar marks indicating errors are no longer rendered in the wrong color (#16106)
* The issue where stable-branded binaries contained a `-preview` moniker has been fixed (#16122)
* The system menu no longer grows hundreds of copies of the `Settings...` menu item when you close and reopen a window (#16225)
* conhost: rectangular copy with the <kbd>Alt</kbd> key no longer merges text onto one line (#16197)

_Binary files inside the unpackaged distribution archive bear the version number `1.19.231113002-preview`._

---

### Asset Hashes

- Microsoft.WindowsTerminalPreview_1.19.3172.0_8wekyb3d8bbwe.msixbundle
   - SHA256 `28B9224679ED6CA37D42E613E3AF5A64D4D7D28F940A5C6BD30D3D3BD076E0DA`
- Microsoft.WindowsTerminalPreview_1.19.3172.0_arm64.zip
   - SHA256 `6554CE37BA8DFEEABFDE8A499B51041210B8BC904B01BAEC491F97D915F09130`
- Microsoft.WindowsTerminalPreview_1.19.3172.0_x64.zip
   - SHA256 `BF255F7DECF7FB2C70E93FC87AD713DAE81FAA94FE8CF49B0C1B842D3609809D`
- Microsoft.WindowsTerminalPreview_1.19.3172.0_x86.zip
   - SHA256 `21A8F63FB5F65757172C1144552B4C1CA4AA2E9A72BDB072B836265731338CF6`

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.19.3172.0'>Windows Terminal Preview v1.19.3172.0</a>.</em>

**Comment by farzadoxo** [comment]
The first the best the mest

**Comment by jessey-git** [comment]
The `winget` update needs punched in the face: https://github.com/microsoft/winget-pkgs/pull/126840

**Comment by RobDover** [comment]
What's the plan for releasing this version for real? Do you know when you're expecting it to be shipped? Looking forward to being able to send commands to all tabs, but I can't take the preview version...
